subscriber/podaac_data_subscriber.py

```diff
@@ -221,24 +226,50 @@ def run():
     ## Sentinel-6 MF datasets also have *.bufr.bin, *.DBL, *.rnx, *.dat
     extensions = args.extensions
 
-    data = args.outputDirectory
-    #You should change `data` to a suitable download path on your file system.
-
-
-    bounding_extent=args.bbox
-    #Change this to whatever extent you need. Format is W Longitude,S Latitude,E Longitude,N Latitude
+    #Error catching for output directory specifications
+    #Must specify -d output path or one time-based output directory flag
```

Comment:
The data directory can always be specified regardless of the time-based output directory. the time-based directory is in addition to to the data directory, so that a user has more control over where output data is placed.

for example, the suer can specify -d as /path/to/my/data/collectionShortName and the -dydoy flag to end up with:

/path/to/my/data/collectionShortName/{year}/{doy}/file

Reply:
Ok, I have changed -d back to being required and allowed for one of the new 3 flags (`-dc`, `-dydoy`, `-dymd`) to be specified in addition.  I have also setup the `.update` file to be saved at the -d destination folder for future checks.

If more than one of the new flags is specified at once, this error is returned:
`podaac_data_subscriber.py: error: Too many output directory flags specified, Please specify exactly one flag from -dc, -dydoy, or -dymd`

If the -dc flag is specified but no cycles are found within the requested collection this error is returned:
`podaac_data_subscriber.py: error: No cycles found within collection granules. Specify an output directory or choose another output directory flag other than -dc.`